components/network/sslh/sslh.p5m

```diff
@@ -0,0 +1,27 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 John Smith. All rights reserved.
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license COPYING license=GPLv2
+
+file Solaris/sslh.xml path=lib/svc/manifest/network/sslh.xml
+file Solaris/sslh.sh path=lib/svc/method/sslh
+file path=usr/sbin/sslh
+file path=usr/share/man/man8/sslh.8
```

Comment:
It depends. If service can reasonably work with this file, I'd keep it in /etc. If it needs customizing in most cases, I'd leave it in /usr/share/doc/sslh/.